protobufs/rpc/sip.proto

```diff
@@ -101,13 +101,19 @@ message InternalCreateSIPParticipantRequest {
 
   livekit.SIPMediaEncryption media_encryption = 28;
 
-  // NEXT ID: 29
+  // Wait for the call to dial before returning.
+  bool wait_until_dialed = 29;
```

Comment:
on second thought. I think the name of this parameter could be confusing.. it reads as if the function will return as soon as we've dialed it..

I think the intention is to say, wait until the creation is complete? (i.e. either success or failure).. so should we call this `wait_until_complete` instead?

or `block`?

Reply:
The current behavior is "wait until LK participant is created" and with the flag set it will be "wait until call is dialed".

So I was thinking that `wait_until_dialed` reflects it quite well, but could rename to `wait_until_complete` as well.

`block` on the other hand is misleading - we already blocks until participant is created. I think we need to specify exactly what is the event we'll wait for.

Would `wait_until_call_pickup` be better maybe?

Reply:
Changed to `wait_until_answered`